src/System.Runtime.WindowsRuntime/src/System.Runtime.WindowsRuntime.csproj

```diff
@@ -25,22 +25,19 @@
   </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Windows_Debug|AnyCPU'" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Windows_Release|AnyCPU'" />
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'uap101aot_Debug|AnyCPU'" />  
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'uap101aot_Release|AnyCPU'" />  
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'uap101aot_Debug|AnyCPU'" />
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'uap101aot_Release|AnyCPU'" />
   <ItemGroup Condition="'$(TargetGroup)'==''">
     <TargetingPackReference Include="System.Private.CoreLib" />
     <ProjectReference Include="$(SourceDir)\mscorlib.WinRT-Facade\mscorlib.WinRT-Facade.csproj" />
     <!-- Needed for the compiler to resolve IObservableMap.MapChanged. -->
     <ProjectReference Include="..\..\System.Runtime\src\System.Runtime.csproj" />
-    <ProjectReference Include="..\..\System.Runtime.Extensions\src\System.Runtime.Extensions.csproj">
-       <Aliases>System_Runtime_Extensions</Aliases>
+    <ProjectReference Include="..\..\System.Runtime.Extensions\ref\System.Runtime.Extensions.csproj">
```

Comment:
The type clashing is the issue that pushes towards using src- where it is most practical for things that reference both S.R and CoreLib. With so much in S.R.E now it would be beneficial to have src here as well, but some of the types don't exist in the UWP CoreLib yet (notably MemoryStream as mentioned). Using ref won't actually run, but it is good enough to get by the compile step for now (albeit with aliases).